modules/core/src/main/scala/Json.scala

```diff
@@ -0,0 +1,134 @@
+package scalaz
+
+package schema
+
+import scalaz.Scalaz._
+import scalaz.schema.JsonSchema._
+
+object Json {
+  type JSON = String
+
+  val module = new SchemaModule {
+    type Prim[A]       = JsonSchema.Prim[A]
+    type ProductTermId = String
+    type SumTermId     = String
+  }
+
+  trait ToJson[S[_]] {
+    def serializer: S ~> (? => JSON)
+  }
+
+  implicit val toJson: ToJson[module.Prim] = new ToJson[module.Prim] {
+    override def serializer: module.Prim ~> (? => JSON) = new (module.Prim ~> (? => JSON)) {
+      override def apply[A](fa: module.Prim[A]): A => JSON = fa match {
+        case JsonString => a => s""""$a""""
```

Comment:
My gut feeling is that `"\""+d+'\"'` will be faster, maybe it's worth to resort some lower level code to make it fast. This won't be code that executes occasionally.

Reply:
@josipgrgurica  Regarding `Applicative` vs `MonadError` is there a way to abstract over this. I can imagine cases where I don't want to evaluate all errors for performance reason and reject an entry based on the first error alone. Also doesn't it need something like `ApplicativeError` I know it from cats but can't find it in sclaz. As for the deduplication in core tests. maybe a good old `util` package with some static test data like the Person schema?


@LGLO I don't know how the internals on string interpolation work. can somebody weigh in on this? I'll be happy to change it